experimental/rules-keeper/proto/ruleset.proto

```diff
@@ -0,0 +1,175 @@
+syntax = "proto3";
+
+package ruleset;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/bazel-contrib/SIG-rules-authors/experimental/rules-keeper/proto;rulesetpb";
+
+message Ruleset {
+  // github_repo_full_path is the full path to its github repo. eg.
+  // bazelbuild/bazel-skylib
+  string github_repo = 1;
+
+  // maintainers of the ruleset in the form of mailbox format as specified in
+  // RFC 5322.
+  repeated string maintainers = 2;
+
+  // update_time is the time when this data is updated.
+  google.protobuf.Timestamp update_time = 3;
+
+  // license of the project.
+  string license = 4;
+
+  // tags are versions of this ruleset
+  repeated Version versions = 5;
+
+  // project_health is the stats on if this project is on the right track.
+  ProjectHealth project_health = 6;
+
+  // project_popularity is the stats on how popular this project is.
+  ProjectPopularity project_popularity = 7;
+
+  // project_activity is the stats on how active this proejct is developed
+  // and  maintained.
+  ProjectActivity project_activity = 8;
+}
+
+// Version denotes a version of a ruleset.
+message Version {
+  // ref is a git reference that could be resolved to a commit object,
+  // normally a tag, ie. a semver starting with v. HEAD stands
+  string ref = 1;
+
+  // SHA of the commit being tagged.
+  string sha = 2;
+
+  // description is the content of the README file.
+  string readme = 3;
+
+  // yanked denotes if this version is yanked.
+  bool yanked = 5;
+
+  // releases made at this version.
+  repeated Release releases = 6;
```

Comment:
When would a ruleset version have more than one release?

Reply:
Ha, I thought Github allowed that.